test/zns-registry.js

```diff
@@ -32,6 +32,41 @@ describe('ZkBNB Registry Contract', function () {
     await zkbnb.deployed();
   });
 
+    describe('ZNS Registry', function () {
```

Comment:
/ZNS/ZnsRegistry.test.ts and ./zns-registry.js
Why test case of ZNSRegistry has two files?